src/galois.c

```diff
@@ -37,6 +37,7 @@ plank@cs.utk.edu
 # define free(m) kfree(m)
 # define fprintf(m, fmt, args...) printk("%s:%d " fmt, __FUNCTION__, __LINE__, ##args)
 # define GALOIS_EXPORTED(s) EXPORT_SYMBOL(s)
+MODULE_LICENSE("LGPL v2.1");
```

Comment:
As the copyright header says it is "under the terms of the GNU Lesser General Public
License", it seems OK to change this.